pkg/investigations/ccam/ccam.go

```diff
@@ -62,6 +64,22 @@ func Investigate(r *investigation.Resources) (investigation.InvestigationResult,
 	}
 }
 
+func (c *CCAM) Name() string {
+	return "Cluster Credentials Are Missing (CCAM)"
+}
+
+func (c *CCAM) Description() string {
+	return "Detects missing cluster credentials"
+}
+
+func (c *CCAM) CheckActive(alert string) bool {
+	return false
```

Comment:
is this special to ccam because it does not relate to an alert and thus always false?